### ISSUE TITLE
Remove confirmation dialog from update50

### ISSUE THREAD

**Issue by dmalan** [cs50/codespace#47]
To save a step, shall we eliminate

<img width="604" alt="Screen Shot 2022-02-09 at 1 01 56 PM" src="https://user-images.githubusercontent.com/788678/153262020-89d7b41e-c2bf-45f2-91b4-da67112fbef9.png">

which is already a bit confusing since the next dialog has a **Rebuild** button:

<img width="649" alt="Screen Shot 2022-02-09 at 1 02 33 PM" src="https://user-images.githubusercontent.com/788678/153262119-a8962fe9-b30c-4bca-b6f3-366eb4b0899b.png">

But the toast has a **Rebuild Now** button:

<img width="565" alt="Screen Shot 2022-02-09 at 1 02 38 PM" src="https://user-images.githubusercontent.com/788678/153262146-5591ac49-4381-475d-94de-9f6cba46cc49.png">

So our own advice to `Click "Rebuild"` is a bit confusing anyway?

https://github.com/cs50/codespace/blob/main/update50.sh#L33

**Comment by rongxin-liu** [comment]
Should we perhaps keep the first prompt to advise the user to click "Rebuild Now" when prompted? So that we make it clear what to click? We can eliminate the second prompt because it is a bit redundant.

**Comment by dmalan** [comment]
Oh, how can we suppress the second prompt? I thought that was generated by the Codespaces extension?

**Comment by rongxin-liu** [comment]
Yes, because the second prompt was induced by us (we explicitly trigger the rebuild action). Since changes made to the `.devcontainer.json` file will also trigger a prompt for container rebuild, we no longer need to trigger the rebuild action.

[https://github.com/cs50/codespace/commit/151372d44b7e9c9c339a3c0bb579093f1bbb15c3](https://github.com/cs50/codespace/commit/151372d44b7e9c9c339a3c0bb579093f1bbb15c3)
